app/services/sip_arrange.service.js

```diff
@@ -67,9 +72,11 @@
             child.has_children = true;
             child.children = [];
             child.children_fetched = false;
+            child.directory = true;
           } else {
             // file
             child.has_children = false;
+            child.directory = false;
```

Comment:
`.directory` allows easier verifying of directories with no children; that's important since we track arrange files and directories distinctly from each other.